torch/csrc/jit/backends/nnapi/nnapi_backend_preprocess.cpp

```diff
@@ -0,0 +1,48 @@
+#include <pybind11/pybind11.h>
+#include <torch/csrc/jit/backends/backend.h>
+#include <torch/csrc/jit/backends/backend_preprocess.h>
+#include <torch/csrc/jit/python/pybind_utils.h>
+
+namespace py = pybind11;
+
+// Converts model to nnapi and serializes it for mobile
+// Returns a dictionary string with one entry:
+// Key: "NnapiModule"
+// Value: a string of the nnapi module, saved for mobile
```

Comment:
The value should have a clearer description: i.e. what is this string representation?